src/qtrendplot.cpp

```diff
@@ -78,7 +89,25 @@ void QTrendPlot::removeSelectedGraphs(){
 }
 
 void QTrendPlot::addSelectedGraphs(){
-    this->addGraphList(casuTree->selectedItems());
+    QList<QTreeWidgetItem*> selectedList = casuTree->selectedItems();
+
```

Comment:
casuTree items are also dataBuffer where plotting data is stored so it is easy to plot those values directly from selected items.